Cargo.toml

```diff
@@ -12,7 +12,7 @@ version = "0.0.1"
 crate-type = ["lib"]
 
 [patch.crates-io]
-esp-idf-sys = {git = "https://github.com/npmenard/esp-idf-sys.git"}
+#esp-idf-sys = {git = "https://github.com/npmenard/esp-idf-sys.git"}
```

Comment:
I updated my fork of esp-idf-sys, can you try it?

Reply:
seems to be working without any issues.

Reply:
maybe not...
getting this error now when running it, need to look into it further. 
```
thread 'main' panicked at 'unsafe precondition(s) violated: ptr::read requires that the pointer argument is aligned and non-null', ~/.rustup/toolchains/esp/lib/rustlib/src/rust/library/core/src/panicking.rs:126:5
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
thread caused non-unwinding panic. aborting.

abort() was called at PC 0x4026cc7a on core 1
```